TypeError: Cannot create property 'amtWsEventSecret' on string

### Discussed in https://github.com/Ylianst/MeshCentral/discussions/5464
```
ERROR: MeshCentral failed with critical error, check mesherrors.txt. Restarting in 5 seconds...
ERR: Ignoring invalid configuration option passed to Connection: secure. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection
ERR: Ignoring invalid configuration option passed to Connection: secure. This is currently a warning, but in future versions of MySQL2, an error will be thrown if you pass an invalid configuration option to a Connection
MeshCentral HTTP redirection server running on port 82.
MeshCentral v1.1.14, WAN mode.
ERR: /home/pi/node_modules/meshcentral/meshcentral.js:1501
if (obj.dbconfig.amtWsEventSecret == null) { obj.crypto.randomBytes(32, function (err, buf) { obj.dbconfig.amtWsEventSecret = buf.toString('hex'); obj.db.Set(obj.dbconfig); }); }
^
TypeError: Cannot create property 'amtWsEventSecret' on string '{"_id":"dbconfig","version":1,"amtWsEventSecret":"c35540a17c955ab8b69da7db3064f40a3d12036521190da92888a4393f1c8e1a"}'
at RandomBytesJob. (/home/pi/node_modules/meshcentral/meshcentral.js:1501:137)
at RandomBytesJob. (node:internal/crypto/random:114:5)
at RandomBytesJob.onJobDone (node:internal/crypto/random:309:3)
```
Node.js v18.17.1

I'm still getting crazy :-)
As soon as I insert something into the config.json I immediately get Error at module errors again
e.g. this one

`  "agentCustomization": {
        "displayName": "Fernwartung...",
        "description": "Die Fernwartung... von .... ist kostenpflichtig und wird mit XX Euro a Stunde berechnet - Service Tel: 00409....",
        "companyName": "Service .....",
        "serviceName": "fernwartung...",
        "image": "csw_msc.png",
        "installText": "Die Fernwartun...... von ...... und wird mit XX Euro a Stunde berechnet - Service Tel : 0049......",
        "fileName": "FernwartungNM_Agent",
	"foregroundColor": "255,0,0",
	"backgroundColor": "255,255,0"
     },`

The message after starting Meshcentral is then :
`
Error: Cannot find module '/home/pi/meshcentral-data/node_modules/meshcentral'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
Node.js v18.17.1`

I suspect that the config.json has changed completely. And as soon as I activate the old meshcentral-data folder (initially without a database) it reloads modules and no longer starts.
Now suddenly the 2FA finally works again even when I use the new data folder. There must be an easier way to restore the basic settings. The whole Meshcentral is a tough job.

--- a/db.js
+++ b/db.js
@@ -1244,7 +1244,11 @@ module.exports.CreateDB = function (parent, func) {
                     var docs = [];
                     for (var i in results) {
                         if (results[i].doc) {
-                            docs.push(results[i].doc);
+                            if (typeof results[i].doc == 'string') {
+                                docs.push(JSON.parse(results[i].doc));
+                            } else {
+                                docs.push(results[i].doc);
+                            }
                         } else if ((results.length == 1) && (results[i]['COUNT(doc)'] != null)) {
                             // This is a SELECT COUNT() operation
                             docs = results[i]['COUNT(doc)'];

